lib/repositories/local_repository.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter_project/local_repository_response.dart';
+import 'package:flutter_project/models/genres.dart';
+import 'package:flutter_project/repositories/genre_repository.dart';
+import 'package:flutter_project/repositories/movies_repository.dart';
+
+import '../models/movie.dart';
+
+class LocalRepository {
+  MoviesRepository movieRepo = MoviesRepository();
+  GenreRepository genreRepo = GenreRepository();
```

Comment:
You can create a LocalRepository constructor that has movieRepository and also genresRepository

Reply:
Done